JavaScript and Ajax/StudentDeleteRemote.cls

```diff
@@ -0,0 +1,19 @@
+public class StudentDeleteRemote {
+    
+    public List<Id> studentIdList { get; set; }
+    public StudentDeleteRemote() { } // empty constructor
+    
+    @RemoteAction
+    public static boolean deleteStudent(List<Id> studentIdList) {
```

Comment:
`Boolean` data type should start with capital letter

Reply:
changed it.